fronts-client/src/components/form/ChefMetaForm.tsx

```diff
@@ -84,12 +142,46 @@ const Form = ({
   );
 };
 
+const chefMetaToForm = (meta: ChefCardMeta): ChefCardFormData => {
+  const {
+    imageSrc,
+    imageSrcHeight,
+    imageSrcWidth,
+    imageSrcThumb,
+    imageSrcOrigin,
+    bio,
+    ...valuesToCopy
+  } = meta;
+
+  return {
+    ...valuesToCopy,
+    bio: bio ?? '',
+    image: {
+      src: imageSrc,
+      height: strToInt(imageSrcHeight),
```

Comment:
I believe that if the `imageSrcHeight` string is not a valid base-10 number then `parseInt()` (called by `strToInt`) will return `NaN` - is this case handled OK when the ChefCardFormData is serialized?